app/spdb/build.sbt

```diff
@@ -221,32 +222,36 @@ def swalker(version: Version) = AppConfig(
   )
 ) extending List(with_gogo(version), swalker_credentials(version))
 
-// FNUSSBER
-def fnussber(version: Version) = AppConfig(
```

Comment:
As in the previous PR, I don't think we should delete fnussber configuration until we make sure olesja/sabrina can add their own